builder/builder.Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM debian:jessie
+
+RUN apt-get update && \
+    apt-get install -y curl debootstrap bzip2 && \
+    apt-get clean
+
+ENV WORKDIR /var/builder
+WORKDIR ${WORKDIR}
+
+ARG DOCKER_VERSION=1.11.2
+RUN curl -sSLO https://github.com/docker/docker/archive/v${DOCKER_VERSION}.tar.gz && tar -xzf v${DOCKER_VERSION}.tar.gz
```

Comment:
oh right, I missed that you were downloading the source archive, not the binary.

Maybe you could remove the vendored `mkimage-debootstrap.sh` from the repo since you're fetching it from github now?